website/content/docs/integration-guide.md

```diff
@@ -0,0 +1,46 @@
++++
+toc = true
+weight = 3
```

Comment:
That should at least be after using Bloop with sbt and Maven, given this is more devtool-developer oriented.

Reply:
Hm, I'm not sure. I think it's a natural extension of "bloop basics" and I like to see all the main documents in the first row start by 'The' and explain the philosophy behind the project.
  
I believe people should reach the integration guide with sbt and maven because of links in the basics guide, mainly. My main concern is that having the integration guide after all the explanations to integrate with build tool does not encourage reading up everything from the beginning to the end, specific build tools instructions are to be consumed on an individual basis.

Reply:
If this page is for tooling developers to integrate with bloop then I think you should go close to the bottom. Your primary pages should be user focused on how to get up and running with bloop.